web/static/js/study-responses-consent-ruling.js

```diff
@@ -56,7 +57,7 @@ $(document).ready(function () {
 
     function saveComments() {
         let currentText = $responseComments.val();
-        if (currentText) {
+        if (currentText && $currentlySelectedResponse) {
```

Comment:
Correct me if I am wrong, but I think you have call `val()` or check the `length` of a jQuery object to check it's truthiness.

Reply:
That's probably true in some cases, but here I get `undefined` for `$currentlySelectedResponse` when no response is selected from the list on the left. So just check whether or not it's undefined is giving me the result I need here (only check for a comment to save if a response is selected), and trying to call `val()` or `length` on that variable produces an error when no response is selected.